Sources/Valet/MigratableKeyValuePair.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  MigratableKeyValuePair.swift
+//  Valet
+//
+//  Created by Dan Federman on 5/20/20.
+//  Copyright © 2020 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+/// A struct that represented a key:value pair that can be migrated.
+public struct MigratableKeyValuePair<KeyType: Hashable>: Hashable {
```

Comment:
Maybe?

```suggestion
public struct MigratableKeyValuePair<Key: Hashable>: Hashable {
```

https://swift.org/documentation/api-design-guidelines#name-according-to-roles